app/assets/stylesheets/style.scss

```diff
@@ -24,6 +24,12 @@ body {
   font-size: 80%;
 }
 
+.course-info span img {
+    float: left;
+    margin-right: 15px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered float, margin-bottom, margin-right